### ISSUE TITLE
Custom View Support now working

### ISSUE THREAD

**Issue by ramkia613** [phrase/phrase-android#62]
Phrase version: 3.6.1

Custom View is not working as expected. Still, it shows the value from the local string file not from the phrase.

My code
```
val attributes = context.obtainStyledAttributes(attrs, R.styleable.CustomView)
   with(attributes) {
          binding.title.text = getStringWithPhrase(this, R.styleable.CustomView_title)
          binding.value.text = getStringWithPhrase(this, R.styleable.CustomView_value)
      }
```

**Comment by theSoenke** [comment]
Hi @ramkia613, could you check whether the translations has been fetched over-the-air? You could check this with `Phrase.getTranslations().getString("keyname")`

**Comment by ramkia613** [comment]
@theSoenke yes `Phrase.getTranslations().getString("keyname")` it fetches the value from OTA.

**Comment by theSoenke** [comment]
@ramkia613 thanks for checking. We found an issue when overwriting the app delegate that resulted in not returning the OTA translations in custom views. This should now be fixed in version 3.6.2 https://github.com/phrase/phrase-android/releases/tag/3.6.2 Let me know if you should still see anything unexpected
